canal/canal.go

```diff
@@ -458,6 +465,10 @@ func (c *Canal) SyncedPosition() mysql.Position {
 	return c.master.Position()
 }
 
+func (c *Canal) SyncedTimestamp() uint32 {
```

Comment:
where do we use it?

Reply:
That's needed for calculating replication delay, like `delay := time.Now().Unix() - int64(c.SyncedTimestamp())`.
Currently, the only way to get event's timestamp, is to read `event.Header.Timestamp` in the `OnRow` handler. But if some tables are skipped in the canal configuration, there's no way to get timestamp at all until we meet a row from non-skipped table

Reply:
Maybe better  to expose an int like MySQL's seconds_behind_master, calcuated in BinlogSyncer and taking accout into clock diff between canal and master?

Reply:
Personally I am fine about it. But you should probably suggest this change to owners